scripts/contracts/contracts.yml

```diff
@@ -223,3 +223,4 @@ PermissionContracts:
         - VersionManager
       functions:
         - 'setVersion(uint32)'
+        - 'setProtocolVersion(uint32)'
```

Comment:
Please update permission section in genesis module.

Reply:
ok